civic_europe/assets/js/app.js

```diff
@@ -14,9 +13,10 @@ require('../../../civic_europe/assets/js/civic_europe')
 
 $(function () {
   widget.initialise('a4', 'comment', ReactComments.renderComment)
-  widget.initialise('a4', 'comment_async', ReactCommentsAsync.renderComment)
```

Comment:
this might be the cause for missing `watch` and `support` buttons
should close #552

Reply:
I'll check this on dev, I have database issues